packages/api/tests/profile-moderation.test.ts

```diff
@@ -0,0 +1,46 @@
+import { moderateProfile } from '../src'
+import type {
+  ModerationBehaviors,
+  ModerationBehaviorScenario,
+} from '../definitions/moderation-behaviors'
+import { ModerationBehaviorSuiteRunner } from './util/moderation-behavior'
+import { readFileSync } from 'fs'
+import { join } from 'path'
+
+const suite: ModerationBehaviors = JSON.parse(
+  readFileSync(
+    join(__dirname, '..', 'definitions', 'profile-moderation-behaviors.json'),
+    'utf8',
+  ),
+)
+
+const suiteRunner = new ModerationBehaviorSuiteRunner(suite)
+
+describe('Post moderation behaviors', () => {
+  const scenarios = Array.from(Object.entries(suite.scenarios))
+  it.each(scenarios)(
+    '%s',
+    (_name: string, scenario: ModerationBehaviorScenario) => {
+      const res = moderateProfile(
+        suiteRunner.profileScenario(scenario),
+        suiteRunner.moderationOpts(scenario),
+      )
+      expect(res.account).toBeModerationResult(
+        scenario.behaviors.account,
+        'account',
+        JSON.stringify(res, null, 2),
+      )
+      expect(res.profile).toBeModerationResult(
+        scenario.behaviors.profile,
+        'profile content',
+        JSON.stringify(res, null, 2),
+      )
```

Comment:
The type of `behaviors` here comes from `ModerationBehaviorScenario`, which doesn't have they keys `behaviors.account` or `behaviors.profile`.  It just has the following: `behaviors.content`, `behaviors.avatar`, and `behaviors.embed`.  Not 100% sure whether the tests or types need updating, but I expect there's a little something to tidy here.

Reply:
Taking a closer look, I expect it's just that the `ModerationBehaviorScenario` type needs to be updated 👍